### PR TITLE
Getblobs2

### PR DESCRIPTION
**Pull request by MangyCoyote**

### CODE REVIEW

File: src/azmi-commandline/azmi-commandline.cs [hunk 1]
```diff
@@ -5,10 +5,21 @@
 using System.CommandLine.Invocation;
 
 namespace azmi_commandline
-{
-
+{    
     class Program
     {
+        class getBlobsOptionsType
```
Comment (iricigor) [inline src/azmi-commandline/azmi-commandline.cs]: Please, add some TODO: comment here. Unless cleaned up, this will bite us in future.

File: src/azmi-commandline/azmi-commandline.cs [hunk 2]
```diff
@@ -201,7 +244,7 @@ static RootCommand ConfigureArguments()
             Operations operations = new Operations();
 
             // gettoken
-            getTokenCommand.Handler = CommandHandler.Create<string, string, bool, bool>((endpoint, identity, verbose, JWTFormat) =>
+            getTokenCommand.Handler = CommandHandler.Create<string, string, bool, bool>((endpoint, identity, JWTFormat, verbose) =>
```
Comment (iricigor) [inline src/azmi-commandline/azmi-commandline.cs]: Is this change intended or just our test?

Comment (MangyCoyote) [reply src/azmi-commandline/azmi-commandline.cs]: Intended. Order of parameters should go from most important ones to least important in methods.
